### ISSUE TITLE
Android Quicklook support without an external application?

### ISSUE THREAD

**Issue by pranavrajsinghchauhan** [philipphecht/react-native-doc-viewer#23]
Is it supported in current version? If not can you please share tentative timeline of the availability

**Comment by philipphecht** [comment]
No. Sorry I can't give you a timeline because at the moment I have other things to do. But feel free to send me a pull request. Remember this is a open source project. I can say i will release this feature.

**Comment by pranavrajsinghchauhan** [comment]
👍 @philipphecht: in case you have done any research for Android and Windows support, can you please provide me the references, I will take it from there

Meanwhile I will also do the required research

**Comment by Noitidart** [comment]
Very interested in this as well. I wanted to move to this instead of ![react-native-pdf-view](https://github.com/cnjon/react-native-pdf-view) as that only suppots pdf.

**Comment by bradleySuira** [comment]
Hi Guys, in Android i think that it's not posible, check this: [](https://stackoverflow.com/questions/20330711/what-is-the-counterpart-of-ios-qlpreviewcontroller-or-uidocumentinteractioncont/20438754)

It's only possible if you write your own viewers. The iOS app is calling out to an external class which just happens to render inside your own view. Not much different to Android except, you use an intent rather than call a specific app or framework which is more flexible since the user chooses how the file is opened.
